### ISSUE TITLE
Problem with handling polish locale

### ISSUE THREAD

**Issue by qunerd** [arca-computing/MultipleDatePicker#80]
Steps to reproduce:

1. Include Moment.js
2. Include moment/locale/pl.js - Moment.js will automagically use this locale now.
3. Problem occurs:

```
Cannot read property '10' of undefined
    at getMonthYearToDisplay (multipleDatePicker.js:177)
    at Scope.scope.generate (multipleDatePicker.js:438)
    at multipleDatePicker.js:243
    at Scope.$digest (angular.js:17524)
    at Scope.$apply (angular.js:17790)
    at angular.js:19621
    at completeOutstandingRequest (angular.js:5964)
    at angular.js:6243
```

I've tried printing current date:
```
moment().format('MMMM Do YYYY, h:mm:ss a')
```
worked without any problems and displayed the date in polish.

**Comment by mgohin** [comment]
Found it, strange for polish there is no `_months` on the localData.
But it doesn't matter, I should have done it better since day 1.
Pushing new release in 2min ;)

**Comment by mgohin** [comment]
Available in 2.1.1, let me know if it's ok ;)

**Comment by qunerd** [comment]
Works like a charm, thanks for the quick release!
